swapi_api/view/Home.vue

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
folder view i views do wywalenia - nie miało być takiej struktury. Mamy modules i tam tworzysz folder homeModule i tam to wrzucasz